### ISSUE TITLE
[bug] system overload & load percentage

### ISSUE THREAD

**Issue by mattia-b89** [konkor/cpufreq#145]
If system reach the "overload" status (which turns red the extension)  
once the system load lowers

Steps to reproduce it:
- feed your CPU so it has some load
- reduce working CORE to only one
- system overload
- restore working CORE number as before
- system stays overloaded even if it not true

I attach a screenshot where you can easily notice system load is normal (the blue graph on the left of the purple-yellow one)  
but system load percentage is crazy, such as _billions_ % and _overload_ , as well

![Screenshot from 2020-08-02 11-14-29](https://user-images.githubusercontent.com/545463/89119939-48de0b00-d4b2-11ea-9bd7-3cbd52e38905.png)

**Comment by debian-user-france1** [comment]
whata f- wt-...
Thats crazy... I would try that..

**Comment by debian-user-france1** [comment]
on one core is "normal" load 70 percent...
my ram is over-bitcoinerd... 120 sys load
138 system load
goes down to 127
my disk stopped (HDD)
lets start libre office, NO lets start Bitcoin Core bitcoin wallet wich isn't updatet right, so MUCH cpu load.. lets try...
WT- 80% memory of 8 GB RAM
212 load, very slow
notice: I have irqbalance...
gimp is awesome for loading..
370 load, gimp isn't started...
gimp started, 444 load, lets do only 50 percent cpu ... Oh, I have 3 ghz...
bitcoin core started
400 % load...
lets do updates..
I does already, but sudo apt update is great... no effect..
I does to only 28 percent frequency..

fakt: can't bring over 440 load, what happened, wich ubuntu version do u have?
I have 20.04 LTS

**Comment by debian-user-france1** [comment]
my computer is finely regenerating now.. load is sinking..

**Comment by debian-user-france1** [comment]
I think, it was a bug of the program, and I DON'T KNOW HOW YOU OVERLOADED YOUR CPU SO MUCH

**Comment by konkor** [comment]
Yeah, looks weird but System Loading could be bigger then 100% unlike CPU Loading/Utilization.

Also you should check your settings and disable battery event if it's enabled. It could be system races for the event if you are using another power management or even a few.

**Comment by debian-user-france1** [comment]
Yes, but my maximum overload is with a Virtualbox only under 600

**Comment by debian-user-france1** [comment]
You had some billions...

**Comment by konkor** [comment]
@debian-user-france1 yeah, it's weird. Can you open a terminal when that happens and get output of:
```sh
cat /proc/cpuinfo|grep processor|wc -l
cat /proc/loadavg
```
Thanks!

**Comment by debian-user-france1** [comment]
I think I can bring that up with a virtual machine...

**Comment by debian-user-france1** [comment]
You have to ask mattia-b89

**Comment by mattia-b89** [comment]
I will post the output as soon as possible
Thx @konkor for your work!

**Comment by mattia-b89** [comment]
I am back! I stress the CPU and system's load get crazy

```
Every 2.0s: cat /proc/cpuinfo|grep proces...

1

Every 2.0s: cat /proc/loadavg

15390.56 4656.96 1655.16 1/980 73055
```

for sake of completeness I my CPU using *stress* utility:
`stress --cpu 8 --io 4 --vm 2 --vm-bytes 128M --timeout 100s`
